src/libs/humanizer/modules/sushiSwapModule.ts

```diff
@@ -19,9 +19,7 @@ export const sushiSwapModule: HumanizerCallModule = (
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   options?: any
 ) => {
-  const routeProcessorIface = new Interface(
-    Object.values(getKnownAbi(humanizerMeta, 'RouteProcessor', options))
```

Comment:
This module isn't working properly
![image](https://github.com/AmbireTech/ambire-common/assets/68795596/4f3b576f-2014-4683-aa25-25b18a872d59)